org.mixedrealitytoolkit.input/InteractionModes/NearInteractionModeDetector.cs

```diff
@@ -16,63 +16,99 @@ namespace MixedReality.Toolkit.Input
     [AddComponentMenu("MRTK/Input/Near Interaction Mode Detector")]
     public class NearInteractionModeDetector : ProximityDetector
     {
+        /// <summary>
+        /// The set of near interactables that belongs to near interaction
+        /// </summary>
         [SerializeField]
-        [Tooltip("The set of near interactors that belongs to near interaction")]
-        private List<XRBaseInteractor> nearInteractors;
+        [Tooltip("The set of near interactables that belongs to near interaction")]
+        private List<XRBaseInteractable> nearInteractables;
 
         /// <summary>
-        /// Used to keep track of the previously detected colliders so that we can know which
-        /// colliders stopped being detected and update their buttons front plate and rounded
-        /// rect if they are labeled as dynamic (based on proximity).
+        /// Used to keep track of the previously detected interactrables so that we can know which
+        /// interactable stopped being detected and update their buttons front plate RawImage.
         /// </summary>
-        List<Collider> previouslyDetectedColliders = new List<Collider>();
+        private HashSet<IXRProximityInteractable> previouslyDetectedColliderInteractableDuples = new();
```

Comment:
No longer a "Duple"

Reply:
oops, I missed that.  Fixed in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/659/commits/a158aa6824f0921963dd143379e87db115469969 resolving this.